src/modules/destination.js

```diff
@@ -33,4 +33,21 @@ const initializeTrip = async (props) => {
   }
 };
 
-export { getCoords, initializeTrip };
\ No newline at end of file
+const addActivity = async (activity, actTimes, tripId) => {
+  try {
+    const response = await axios({
+      method: "PUT",
+      url: "api/v1/activities",
+      params: {
+        tripId: tripId,
+        activity: activity,
+        actTimes: actTimes
```

Comment:
make a POST request to `api/v1/activity_types`, params { trip, activity_type, actTimes(or whatever) }

this is valid until I change my mind

Reply:
done